lib/seed/user_seeder.rb

```diff
@@ -11,15 +13,36 @@ def initialize(config:)
       @logger = Rails.logger.child(class: self.class.name)
       @number_of_users_per_facility = config.max_number_of_users_per_facility
       @organization = Seed.seed_org
-      puts "Starting #{self.class} with #{config.type} configuration"
+      announce "Starting #{self.class} with #{config.type} configuration"
     end
 
     attr_reader :config
     attr_reader :logger
     attr_reader :number_of_users_per_facility
     attr_reader :organization
+    delegate :stdout, to: :config
 
     def call
+      create_dashboard_admins
+      create_mobile_users
+    end
+
+    private
+
+    def create_dashboard_admins
+      unless EmailAuthentication.exists?(email: "power_user@simple.org")
+        FactoryBot.create(:admin, :power_user, full_name: "Power User", email: "power_user@simple.org", password: config.admin_password)
+      end
+
+      unless EmailAuthentication.exists?(email: "cvho@simple.org")
+        fg_1, fg_2 = *FacilityGroup.take(2)
+        user = FactoryBot.create(:admin, :manager, full_name: "CVHO", email: "cvho@simple.org")
+        user.accesses.create! resource_id: fg_1.id, resource_type: "FacilityGroup"
+        user.accesses.create! resource_id: fg_2.id, resource_type: "FacilityGroup"
+      end
+    end
```

Comment:
As Claudio had recommended, can we add a few additional admins here, please?

* STS - manage access to one facility group (or one block)
* District official - view-reports-only access to one district
* Medical officer - view-all access to once facility

Reply:
Yup, added those. 👍